src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -262,7 +277,7 @@ function MultiCombobox<Item>({
               <InputControl
                 invalid={invalid}
                 disabled={disabled}
-                variant={isOpen && items.length ? 'solid' : 'outline'}
+                variant={multiComboboxVariant}
```

Comment:
Should we still account for the `items.length`?

Reply:
Nice catch!